internal/traffic_shaping.py

```diff
@@ -98,13 +99,20 @@ def configure(self, job, task):
         if 'shaperLimit' in job:
             shaperLimit = self._to_int(job['shaperLimit'])
         if self.shaper is not None:
-            # If a lighthouse test is running, force the Lighthouse 3G profile:
+            # If a lighthouse test is running, force the Lighthouse 3G profile for mobile
+            # or 4G for desktop:
             # https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md
-            # 1.6Mbps down, 750Kbps up, 150ms RTT
             if task['running_lighthouse'] and not job['lighthouse_throttle']:
-                rtt = 150
-                in_bps = 1600000
-                out_bps = 750000
+                if self.options.android or ('mobile' in job and job['mobile']):
+                    # 1.6Mbps down, 750Kbps up, 150ms RTT
+                    rtt = 150
+                    in_bps = 1600000
+                    out_bps = 750000
+                else:
+                    # 10Mbps, 40ms RTT
+                    rtt = 40
+                    in_bps = 10240000
+                    out_bps = 10240000
```

Comment:
Should we do it even when `self.job['dtShaper']`is true? https://github.com/catchpoint/WebPageTest.agent/blob/4ac1576773d201c97af548723ae277432a172953/internal/devtools_browser.py#L856

Reply:
Ahh, yes - good catch.  Update inbound.